docs/account-ref.rst

```diff
@@ -25,3 +25,12 @@ Role
 
 .. autoclass:: canvasapi.account.Role
     :members:
+
+=====
+Login
```

Comment:
I see you have also added a similar section in the new file login-ref.rst
Since the Login class is in its own file, login-ref.rst is the correct place for it.
Remove this section.